benchmarks/user/README.md

```diff
@@ -0,0 +1,233 @@
+# Benchmarking a Devito Operator
+
+## Running benchmarks
+
+`benchmark.py` implements a minimalist framework to evaluate the performance of
+a Devito Operator while varying
+
+* the problem size (e.g., shape of the computational grid)
+* the discretization (e.g., space- and time-order of the input/output fields),
+* the simulation time,
```

Comment:
e.g. number of timesteps?

Reply:
in fact no, only the simulation time in milliseconds con be provided, from which the number of timesteps will be derived

Reply:
In millisecond

Reply:
right, fixing